package.json

```
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "lerna exec --parallel -- babel src -d dist --ignore '**/*.test.jsx,**/*.stories.jsx' --root-mode upward",
     "clear_styled_components": "for d in ./packages/*/*; do rm -rf $d/node_modules/styled-components; done; echo \"Cleared nested packages/**/node_modules/styled-components\"",
+    "deploy-storybook": "storybook-to-ghpages",
```

I needed to add the source branch since it assumes it will be `master`: 
```suggestion
    "deploy-storybook": "storybook-to-ghpages  --source-branch latest",
```